1.8/administration/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -61,7 +61,7 @@ The following instructions demonstrate how to use Docker with OverlayFS on CentO
     $ sudo tee /etc/yum.repos.d/docker.repo <<-'EOF'
     [dockerrepo]
     name=Docker Repository
-    baseurl=https://yum.dockerproject.org/repo/main/centos/$releasever/
+    baseurl=https://yum.dockerproject.org/repo/main/centos/7/
```

Comment:
I tried that and it got replaced with `7server` or something like that which resulted in a broken URL. The instructions are very CentOS7 specific anyways so I didn't see a problem with this fix.